bitnami/timescaledb-bitnami-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# We have to use the bitnami configuration variable to add timescaledb to
+# shared preload list, or else it gets overwritten.
+if [ -z "$POSTGRESQL_SHARED_PRELOAD_LIBRARIES" ]
+then
+    POSTGRESQL_SHARED_PRELOAD_LIBRARIES=timescaledb
+else
+    POSTGRESQL_SHARED_PRELOAD_LIBRARIES="$POSTGRESQL_SHARED_PRELOAD_LIBRARIES,timescaledb"
+fi
+export POSTGRESQL_SHARED_PRELOAD_LIBRARIES
+
+# Fall through to the original entrypoint.
+/opt/bitnami/scripts/postgresql/entrypoint.sh "$@"
```

Comment:
Should this be an `exec` instead?

Reply:
Yes, you're right, this wrapper script shouldn't change PID 1. I'll fix it.

Reply:
https://github.com/timescale/timescaledb-docker/pull/206